app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -252,6 +323,232 @@ async def _date_intensity_symbology(
     return [intensity_job, *merge_jobs], dst_uri
 
 
+async def _merge_intensity_and_date_conf_multi_8(
+    dataset: str,
+    version: str,
+    pixel_meaning: str,
+    date_conf_uri: str,
+    intensity_uri: str,
+    zoom_level: int,
+    parents: List[Job],
+) -> Tuple[List[Job], str]:
+    """Create RGB-encoded raster tile set based on date_conf and intensity
+    raster tile sets."""
+    # import numpy as np
+
+    # Plausible raw values, bands as seen when creating the raster tile set
+    # A = np.ma.array([30080, 20060, 21040, 0], dtype=np.uint16)
+    # B = np.ma.array([1, 1, 2, 0], dtype=np.uint8)
+    # C = np.ma.array([140, 200, 1000, 0], dtype=np.uint16)
+    # D = np.ma.array([31040, 21040, 20040, 0], dtype=np.uint16)
+
+    # The new encoding can be summarized as
+    # value = 2 * DAY + (1 if low confidence else 0)
+    # So if matrix X is in the new encoding,
+    # DAY = X >> 1 and
+    # CONFIDENCE = X & 1 where a CONFIDENCE value of 1 = low, and 0 -= high
+
+    # Anyway, this the calc function I specify when creating the raster
+    # tile set, resulting in the A, B, and C seen coming into this function,
+    # which are in the new encoding.
+    # A, B, C = np.ma.array([
+    #     (A>=20000) * (2 * (A - (20000 + (A>=30000) * 10000)) + (A<30000) * 1),
+    #     (B>0) * (2 * (C + 1461) + (B<2) * 1),
+    #     (D>=20000) * (2 * (D - (20000 + (D>=30000) * 10000)) + (D<30000) * 1)
+    # ], dtype=np.uint16)
+
+    # Also at this point we have the new intensity bands
+    # D = np.ma.array([50, 21, 0, 16], dtype=np.uint8)
+    # E = np.ma.array([55, 34, 16, 20], dtype=np.uint8)
+    # F = np.ma.array([0, 15, 0, 45], dtype=np.uint8)
+
+    # This is how we want the final channels encoded
+    # RED = "(DAY / 255)"
+    # GREEN = "(DAY % 255)"
+    # BLUE = "(((CONFIDENCE + 1) * 100) + INTENSITY)"
+    # ALPHA = First 2 bits GLAD confidence, then 2 bits for GLAD-S2 confidence,
+    #         then 2 bits for RADD confidence, then 2 unused bits.
+    #         Confidences should be a vaue of 2 for high, 1 for low, 0
+    #         for not detected.
+
+    # But we've got three systems, and we want the minimum (first detection)
+    # in any. np.minimum doesn't exclude masked values, so we have to replace
+    # them with something bigger than our data... and then re-mask the
+    # previously masked values afterwards. So unless I'm mistaken, just to
+    # get the minimum of the three alert systems requires something like this:
+    # Oye. What a monster...
+
+    FIRST_ALERT = "(np.ma.array((np.ma.array(np.minimum(A.filled(65535), np.minimum(B.filled(65535), C.filled(65535))), mask=(A.mask & B.mask & C.mask)).filled(0)), mask=(A.mask & B.mask & C.mask)))"
```

Comment:
definitely could use a multiline string here too

Reply:
why are all the vars in this function in constant uppercase format?

Reply:
Eh, good question. Fixed!